src/tunnel.js

```diff
@@ -1,35 +1,36 @@
-var ngrokLauncher = require('screener-ngrok');
-var Promise = require('bluebird');
-var url = require('url');
-var sauceConnectLauncher = require('sauce-connect-launcher');
-var sauceConnection;
+const ngrokLauncher = require('screener-ngrok');
+const Promise = require('bluebird');
+const url = require('url');
+const Saucelabs = require('saucelabs').default;
+
+let sauceConnection;
 
 exports.connect = function({ ngrok, sauce }, tries = 0) {
   if (sauce) {
     return new Promise(function(resolve, reject) {
```

Comment:
As `saucelabs` package returns a promise, can `new Promise` be removed?

Reply:
Done